nbproject/project.properties

```diff
@@ -43,6 +43,10 @@ dist.jar=${dist.dir}/TotalFreedomMod.jar
 dist.javadoc.dir=${dist.dir}/javadoc
 endorsed.classpath=
 excludes=
+file.reference.BukkitTelnet.jar=C:\\Users\\ann\\Desktop\\Coding stuff\\BukkitTelnet.jar

```

Comment:
@LegendIsAwesomes Please uncommit all changes made to this file. You probably should learn how NetBeans' library function works.